package.json

```diff
@@ -106,6 +106,7 @@
   },
   "husky": {
     "hooks": {
+      "pre-commit": "lint-staged"
```

Comment:
what does it do?

Reply:
Was removed before by mistake. It checks for eslint changes before commting, so the commit can be reverted.